packages/paste-website/stories/InlineCode.stories.tsx

```diff
@@ -2,23 +2,20 @@ import * as React from 'react';
 import {Anchor} from '@twilio-paste/anchor';
 import {Box} from '@twilio-paste/box';
 import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
-
-import {InlineCode} from '../src/components/Typography';
+import {InlineCode} from '@twilio-paste/inline-code';
 
 export const RegularInline = (): React.ReactNode => <InlineCode>string</InlineCode>;
 
 export const AnchorInline = (): React.ReactNode => (
-  <InlineCode>
-    <Anchor href="#">string</Anchor>
-  </InlineCode>
+  <Anchor href="#">
```

Comment:
question (non-blocking): why the inversion?

Reply:
Our inline code component only allows strings. Which honestly makes sense. So to stop typescript complaining about it, I have to wrap an inline code with a link.